pkg/controllers/dynakube/extension/eec/statefulset.go

```diff
@@ -266,12 +297,29 @@ func setVolumes(dk *dynakube.DynaKube) func(o *appsv1.StatefulSet) {
 				},
 			},
 			{
-				// TODO: is a configMap.name: eec-runtime-configuration needed?
 				Name: configurationVolumeName,
 				VolumeSource: corev1.VolumeSource{
 					EmptyDir: &corev1.EmptyDirVolumeSource{},
 				},
 			},
+			{
+				Name: httpsCertVolumeName,
+				VolumeSource: corev1.VolumeSource{
+					Secret: &corev1.SecretVolumeSource{
+						SecretName: extensionsControllerTlsSecretName,
+						Items: []corev1.KeyToPath{
+							{
+								Key:  "tls.crt",
```

Comment:
you can check my PR and since we need similar file name -> I've introduced it inside constants package constant https://github.com/Dynatrace/dynatrace-operator/pull/3641/files#diff-ef2b91ac39e4d6c2a00f89195b00a94d53df8100981a99e734fdd45458f494c4R21

Reply:
Agree :+1: It could even make sense to have a var in that consts.go file with the whole []corev1.KeyToPath{} pairings, I believe we do the same in other places

Reply:
nope, you can't use custom type as constant.